pytket/extensions/qujax/qujax_convert.py

```diff
@@ -23,6 +23,10 @@
 from pytket import Qubit, Circuit  # type: ignore
 
 
+class NamedArray(jnp.DeviceArray):
+    __name__: str
```

Comment:
Do you want to add an additional constructor here? What is the default value?

Reply:
No lol this was an idea of mine to enable cleaner printing of the "qujax args" but in the end I settled on the simpler approach of just leaving the converted gates as jnp.DeviceArray instances